app/data/filters.py

```diff
@@ -39,6 +47,26 @@ class DriverRecordFilter(RecordFilter):
     created_max = django_filters.IsoDateTimeFilter(name="created", lookup_expr='lte')
     created_by = django_filters.Filter(field_name='created_by', method='filter_created_by')
     weather = django_filters.Filter(field_name='weather')
+    outside_boundary = django_filters.Filter(field_name='geom', method='filter_outside_boundary')
+
+    def filter_outside_boundary(self, queryset, field_name, boundary_uuid):
+        """Filter records that fall outside the specified boundary."""
+        redis_conn = get_redis_connection('boundaries')
+        bounds_hexewkb = redis_conn.get(boundary_uuid)
```

Comment:
I like that, but unfortunately we're using [`django-redis`](https://niwinz.github.io/django-redis/latest/) rather than [`django-redis-cache`](https://django-redis-cache.readthedocs.io/en/latest/api.html), and it doesn't look to me like `django-redis` has equivalent functionality.